src/bytes.rs

```diff
@@ -265,19 +275,48 @@ impl<'a, T: Sized + AsRef<[u8]> + Send + Sync + 'a> From<Arc<T>> for Bytes<'a> {
     }
 }
 #[cfg(feature = "alloc")]
+impl From<alloc::boxed::Box<[u8]>> for Bytes<'_> {
+    fn from(value: alloc::boxed::Box<[u8]>) -> Self {
+        unsafe extern "C" fn release(this: *const (), capacity: usize) {
```

Comment:
```suggestion
        unsafe extern "C" fn release_box_bytes(this: *const (), capacity: usize) {
```

(for nicer debugger interactions)